base/path.jl

```diff
@@ -362,6 +362,13 @@ function normpath(path::String)
     end
     string(drive,path)
 end
+
+"""
+    normpath(path::AbstractString, paths::AbstractString...) -> AbstractString
```

Comment:
```suggestion
    normpath(path::AbstractString, paths::AbstractString...) -> String
```
This will always return a `String` since `joinpath` always produces a `String`.

Reply:
As far as I can tell all methods of `normpath` and `abspath` yield `String` results regardless of `joinpath`.